### ISSUE TITLE
LogbackValue does not implement Context correctly (1.5.7)

### ISSUE THREAD

**Issue by NilsDammenhayn** [qos-ch/logback#844]
Hi everyone,
I just updated logback-classic to 1.5.7 and got this error when building my spring boot application with tomcat:
```
Caused by: java.lang.AbstractMethodError: Receiver class ch.qos.logback.access.tomcat.LogbackValve does not define or inherit an implementation of the resolved method 'abstract java.util.concurrent.locks.ReentrantLock getConfigurationLock()' of interface ch.qos.logback.core.Context.
	at ch.qos.logback.core.joran.GenericXMLConfigurator.processModel(GenericXMLConfigurator.java:218)
	at ch.qos.logback.core.joran.GenericXMLConfigurator.doConfigure(GenericXMLConfigurator.java:178)
	at ch.qos.logback.core.joran.GenericXMLConfigurator.doConfigure(GenericXMLConfigurator.java:123)
	at ch.qos.logback.core.joran.GenericXMLConfigurator.doConfigure(GenericXMLConfigurator.java:66)
	at ch.qos.logback.access.tomcat.LogbackValve.configureAsResource(LogbackValve.java:230)
	at ch.qos.logback.access.tomcat.LogbackValve.startInternal(LogbackValve.java:159)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:164)
	... 118 more
```
When checking the class `LogbackValue` I can see that the method .getConfigurationLock() has the return type `Object` while the interface `Context` describes a `ReentrantLock`. Looks not correct to me (and my IDE 😉)

**Comment by ceki** [comment]
@NilsDammenhayn  Sorry about this bug.

Fixed in [commit b5096c5090b](https://github.com/qos-ch/logback-access/commit/b5096c5090bb5936). Logback-access 2.0.3 was released earlier today.

**Comment by NilsDammenhayn** [comment]
@ceki Thank you very much, everything looks fine now 👍
